src/main/java/edu/ksu/canvas/TestLauncher.java

```diff
@@ -3,6 +3,8 @@
 import java.io.IOException;
 import java.util.List;
 
+import edu.ksu.canvas.interfaces.AssignmentGroupWriter;
```

Comment:
All the changes in here were also just for testing. This class is intended as something anyone can launch upon checkout to verify that their API key and URL and such are working. It can also be used to test new functionality you are developing but those changes should not be committed.